### ISSUE TITLE
Remove style superContainer from CSS 

### ISSUE THREAD

**Issue by lsterling03** [alvarotrigo/fullPage.js#2812]
What is the reason for adding touch-action:none to the fullpage.js container? Touch scrolling does not work on Android chrome or firefox on my site, but I can fix it if I remove touch-action:none from the fullpage.js css.  -OR- I can also fix it if I remove the responsive options from my fullpage.js settings. It just doesn't work if I have both responsive options AND touch-action:none.

I assume touch-action:none must be there for a reason, so I'm afraid to remove it. But I'm not sure how else to fix the scrolling issue on Android because I need the responsive declaration.

Here are my fullpage.js settings. Is there something here that is incompatible with mobile scrolling?

```
$('#superContainer').fullpage({
	menu: '#sidenav .nav',
	scrollBar: true,
	lockAnchors: false,
	scrollingSpeed: 1000,
	responsiveWidth: 992,
	responsiveHeight: 700,
	slidesNavigation: true,
	fitToSection: true,
	fitToSectionDelay: 500,
	afterRender: function() { 

	  // preloader
	  setTimeout(function() {
		  //video.addEventListener("canplay", function() {
			$('body').addClass('loaded');
		  // }, false);
	  }, 800);

	}
});
```

I am using the latest version of fullpage.js from cdnjs.

**Comment by alvarotrigo** [comment]
> Touch scrolling does not work on Android chrome or firefox on my site

Does the [demo page](http://alvarotrigo.com/fullPage/) scrolls correctly on your device?

If so, does the [normal scroll demo](https://alvarotrigo.com/fullPage/examples/normalScroll.html#secondPage) scrolls correctly on your device?

**Comment by lsterling03** [comment]
Scrolling does work properly on the demos. However, I noticed the one difference is that my fullpage wrapper was id="superContainer". That was what was applying the mobile-touch styles to the wrapper. When I switched the id to "fullPage", then it worked fine. What is the deal with using superContainer?

**Comment by alvarotrigo** [comment]
> When I switched the id to "fullPage", then it worked fine. What is the deal with using superContainer?

Interesting... the CSS file has `superContainer` defined in it. That's probably from a super old version of fullpage.js and needs to be removed.

I'll mark it as an enhancement. Thanks for reporting it.

**Comment by lsterling03** [comment]
Awesome. Thanks!

**Comment by alvarotrigo** [comment]
Fixed on dev 2.9.5
